ledger-api/grpc-definitions/com/daml/ledger/api/v1/transaction_service.proto

```diff
@@ -140,3 +143,16 @@ message GetLedgerEndResponse {
   LedgerOffset offset = 1;
 }
 
+message GetLatestPrunedOffsetsRequest {
+  // Empty for now, but may contain fields in the future.
+}
+
+message GetLatestPrunedOffsetsResponse {
+  // The offset up until the ledger has been pruned, without pruning divulgence events.
+  LedgerOffset participant_pruned_up_to_inclusive = 1;
+
+  // The offset up until the ledger has been pruned including pruning all divulgence events.
+  // For more details about all divulgence events pruning,
+  // see PruneRequest.prune_all_divulged_contracts in ``participant_pruning_service.proto``.
```

Comment:
Not sure what does it add except for the extra line..? I would leave it more concise, since I'm not aiming anyway to be exhaustive (i.e. then we would also talk about immediate/retroactive divulgence etc.)

Reply:
Sorry, I made this comment in a hurry.
Yes, I changed the first line slightly and added the second line that matches the wording I suggested in the preceding comment